admin/views/tabs/metas/taxonomies/category-url.php

```diff
@@ -17,7 +15,10 @@
 $yform->light_switch(
 	'stripcategorybase',
 	__( 'Remove the categories prefix', 'wordpress-seo' ),
-	$remove_buttons,
-	false,
-	$stripcategorybase_help
+	[
+		__( 'Keep', 'wordpress-seo' ),
+		__( 'Remove', 'wordpress-seo' ),
```

Comment:
Since the name of this controls is "Remove the categories prefix",  I'd expect "Remove" to be the "on" option. Currently, it's a bit confusing:

<img width="789" alt="Screenshot 2020-03-31 at 10 45 43" src="https://user-images.githubusercontent.com/1682452/78006287-158d4300-733d-11ea-8ab4-a196b103bf58.png">

Alternatively, change the name to "Keep the categories prefix".